driver/main.c

```diff
@@ -1164,18 +1127,32 @@ static long ppm_ioctl(struct file *filp, unsigned int cmd, unsigned long arg)
 		ret = 0;
 		goto cleanup_ioctl;
 	}
-	case PPM_IOCTL_MANAGE_TP:
```

Comment:
Dropped support; ported to follow same logic as `ENABLE/DISABLE_SYSCALL` ioctls.